### ISSUE TITLE
Replying a mail with mu4e doesn't work

### ISSUE THREAD

**Issue by AlejandroCatalina** [syl20bnr/spacemacs#5076]
#### Description
Trying to reply a mail from mu4e (binding `R`) gets `error in process filter: cl-remove-if-not: Symbol's value as variable is void: ´mu4e//search-account-by-mail-address`.

#### Reproduction guide
- Start Emacs
- Open mu4e mode
- Go to any mail
- Press `R` to reply

*Observed behaviour:*
An error message is reported, `error in process filter: cl-remove-if-not: Symbol's value as variable is void: ´mu4e//search-account-by-mail-address`

*Expected behaviour:*
Create and open a new buffer allowing the reply, previously asking for whom to reply to (sender or all recipients)

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.9
- Spacemacs branch: develop (rev. 398bd46)
- Distribution: spacemacs
- Editing style: emacs
- Completion: ivy
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp lispy git markdown org ibuffer ranger
                 (shell :variables shell-default-shell 'eshell shell-default-height 30 shell-default-position 'bottom)
                 spell-checking syntax-checking version-control latex ipython-notebook python java c-c   ruby clojure haskell github eyebrowse semantic erc
                 (elfeed :variables elfeed-feeds
                         '("http://www.howardism.org/index.xml" "http://endlessparentheses.com/atom.xml" "http://www.masteringemacs.org/feed/" "http://emacs-fu.blogspot.com/feeds/posts/default" "http://emacsredux.com/atom.xml" "http://www.lunaryorn.com/feed.atom" "http://emacshorrors.com/feed.atom" "http://swannodette.github.com/atom.xml" "http://batsov.com/atom.xml" "http://twogreenleaves.org/index.php?feed=rss" "https://medium.com/feed/@hlship/" "http://gigasquidsoftware.com/atom.xml" "http://blog.fogus.me/feed/" "http://steve-yegge.blogspot.com/atom.xml" "http://www.rkn.io/feed.xml"))
                 (mu4e :variables mu4e-installation-path "/usr/local/share/emacs/site-lisp/mu4e" mu4e-enable-mode-line t mu4e-enable-notifications t mu4e-account-alist
                       '(("personal"
                         ...)
                         ("college"
                          ...)))
                 spacemacs-ivy)
```

#### Backtrace
I'll edit when I have time
```

```

**Comment by AlejandroCatalina** [comment]
I leave this here in case anyone's suffering the same and got time to fix it, i'll try to address this tonight. As a note, `mu4e//search-account-by-mail` exists as a function in `mu4e/funcs.el`.

**Comment by AlejandroCatalina** [comment]
Okay, fixed in #5077

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
